src/gpio/mod.rs

```diff
@@ -42,14 +42,21 @@ pub struct PinDescription {
     pub board_pin_number: u8,
     bcm_pin_number: Option<u8>,
     pub name: &'static str,
-    options: &'static[PinFunction], // The set of functions the pin can have, chosen by user config
+    pub options: &'static [PinFunction], // The set of functions the pin can have, chosen by user config
+}
+impl std::fmt::Display for PinFunction {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{:?}", self)
+    }
 }
 
 // Model the 40 pin GPIO connections - including Ground, 3.3V and 5V outputs
-pub const GPIO_DESCRIPTION : [PinDescription; 40] = [PIN_1, PIN_2, PIN_3, PIN_4, PIN_5, PIN_6, PIN_7, PIN_8, PIN_9, PIN_10,
-    PIN_11, PIN_12, PIN_13, PIN_14, PIN_15, PIN_16, PIN_17, PIN_18, PIN_19, PIN_20,
-    PIN_21, PIN_22, PIN_23, PIN_24, PIN_25, PIN_26, PIN_27, PIN_28, PIN_29, PIN_30,
-    PIN_31, PIN_32, PIN_33, PIN_34, PIN_35, PIN_36, PIN_37, PIN_38, PIN_39, PIN_40];
+pub const GPIO_DESCRIPTION: [PinDescription; 40] = [
+    PIN_1, PIN_2, PIN_3, PIN_4, PIN_5, PIN_6, PIN_7, PIN_8, PIN_9, PIN_10, PIN_11, PIN_12, PIN_13,
```

Comment:
I probably don't have rust_fmt configured correctly. We should commit code after IDE has rust_formatted it... then the formatting will always be the same.

Reply:
Yaa my IDE does that

Reply:
Mine should too now, so they should coincide.